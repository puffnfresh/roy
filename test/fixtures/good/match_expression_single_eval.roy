data Either a b =
  Left a | Right b

let f = \() ->
  console.log "This should only appear once"
  Right 1

match f()
  case (Left x) = console.log x
  case (Right x) = console.log x
