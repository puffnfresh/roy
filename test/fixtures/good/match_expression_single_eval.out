This should only appear once
1
